{"name": "Проверка счётчиков заказов", "status": "broken", "statusDetails": {"message": "TypeError: 'str' object is not callable", "trace": "self = <tests.test_orders_feed.TestOrdersFeed object at 0x0000027E347AD670>\norders_feed_page = <pages.orders_feed_page.OrdersFeedPage object at 0x0000027E347AEF00>\n\n    @allure.title(\"Проверка счётчиков заказов\")\n    def test_order_counters(self, orders_feed_page):\n>       orders_feed_page.open()\n\ntests\\test_orders_feed.py:22: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <pages.orders_feed_page.OrdersFeedPage object at 0x0000027E347AEF00>, url = None\n\n    def open(self, url: str = None):\n        \"\"\"Открытие страницы. Если url не указан — используется path().\"\"\"\n>       self._url = url if url else self.path()\nE       TypeError: 'str' object is not callable\n\npages\\base_page.py:45: TypeError"}, "start": 1749737380140, "stop": 1749737380140, "uuid": "cbe3664e-3ecd-4f9a-8bc1-855496d22e7c", "historyId": "7ee17e338e716dfe2c3a6db42f1e2c6f", "testCaseId": "7ee17e338e716dfe2c3a6db42f1e2c6f", "fullName": "tests.test_orders_feed.TestOrdersFeed#test_order_counters", "labels": [{"name": "feature", "value": "Лента заказов"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_orders_feed"}, {"name": "subSuite", "value": "TestOrdersFeed"}, {"name": "host", "value": "DESKTOP-05J7DPF"}, {"name": "thread", "value": "22940-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_orders_feed"}]}